### DISCUSSION TITLE
Cant get my github as student!!!

### DISCUSSION THREAD

**Discussion by mason19987** [community/community#55235] [GitHub Education]
### Hi everyone,

I cant get my student benfite, try several times but cant get it?

**Comment by Ntshangase** [comment]
Hello,

It would be easier to assist you if you would share the rejection reason.

**Comment by mason19987** [comment]
Hello Sir
Thank you for responding!

I get this message every time . but i did refreshed my account.
[cid:4cb953f2-c2b0-4646-8639-c109865882ba]
________________________________
Fra: Mntwana ***@***.***>
Sendt: 12. maj 2023 12:20
Til: community/community ***@***.***>
Cc: Safi Meissam ***@***.***>; Author ***@***.***>
Emne: Re: [community/community] Cant get my github as student!!! (Discussion #55235)


Hello,

It would be easier to assist you if you would share the rejection reason.

**Reply by Ntshangase** [reply]
Still a bit unclear, can you provide a screenshot of the error/rejection reason.

**Comment by mason19987** [comment]
I get only this message as i sent to you!

best regards
________________________________
Fra: Mntwana ***@***.***>
Sendt: 12. maj 2023 12:35
Til: community/community ***@***.***>
Cc: Safi Meissam ***@***.***>; Author ***@***.***>
Emne: Re: [community/community] Cant get my github as student!!! (Discussion #55235)


Still a bit unclear, can you provide a screenshot of the error/rejection reason.

**Reply by Ntshangase** [reply]
can you try logging out then in again on your GitHub account, then apply

**Comment by mason19987** [comment]
Yes i can do, i just did it:)
________________________________
Fra: Mntwana ***@***.***>
Sendt: 12. maj 2023 12:39
Til: community/community ***@***.***>
Cc: Safi Meissam ***@***.***>; Author ***@***.***>
Emne: Re: [community/community] Cant get my github as student!!! (Discussion #55235)


can you try logging out then in again on your GitHub account

**Reply by Ntshangase** [reply]
ok, If you still can't get student benefits then i would suggest you try applying from a different device.

**Reply by Ntshangase** [reply]
Hey @mason19987  it seems the you got accepted ?

**Comment by samus-aran** [comment]
Hi @mason19987 

Apologies for the delayed response from the team, we hope you were able to resolve your rejection reasons and receive a successful verdict on your application. I'll be closing this discussion due to lack of recent activity. If you still haven't received your Global Campus benefits please feel free to re-open and I'll be happy to assist!
